### PR TITLE
Scale embedded videos and audios

### PR DESCRIPTION
**Pull request by donmor**
Modified files:
$:/core/modules/parsers/audioparser.js
$:/core/modules/parsers/videoparser.js
Added an attribute to each to make video imported scaled to fit the parent container:
style: {type: "string", value: "width: 100%; object-fit: contain"}

Before:
2019-05-14 12-41-01屏幕截图
2019-05-14 13-06-33屏幕截图

After:
2019-05-14 12-40-15屏幕截图
2019-05-14 13-06-39屏幕截图

### CODE REVIEW

**Comment by Jermolene** [comment]
Hi @donmor many thanks for this, and my apologies for my slow response. It looks good, but I'm afraid we have a small bit of bureaucracy: you need to sign the CLA in a separate PR, as described in [contributing.md](https://github.com/Jermolene/TiddlyWiki5/blob/master/contributing.md).

**Comment by donmor** [comment]
Ok I had just signed the CLA a moment ago

**Comment by donmor** [comment]
Okay I created the CLA PR again to tiddlywiki-com ( #3957 )

**Comment by donmor** [comment]
Hi @Jermolene ,
I just found a PR in my fridge :)
If these changes are helpful, please merge this to the next version.
I'm glad to contribute to this project.
Best regards,
donmor

File: core/modules/parsers/audioparser.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ var AudioParser = function(type,text,options) {
 			type: "element",
 			tag: "audio",
 			attributes: {
-				controls: {type: "string", value: "controls"}
+				controls: {type: "string", value: "controls"},
+				style: {type: "string", value: "width: 100%; object-fit: contain"}
```
Comment (Jermolene) [inline core/modules/parsers/audioparser.js]: Hi @donmor apologies for not noticing before, but it's actually possible to pass the styles directly:

```
tag: "audio",
attributes: {
	controls: {type: "string", value: "controls"}
},
style: {
	width: "100%",
	"object-fit": "contain"
}
```

Comment (donmor) [reply core/modules/parsers/audioparser.js]: So how to pass the styles to the embedded video or audio? For example, I imported a video file, and how to insert it into a tiddler and make it fit the width of the page?

Comment (Jermolene) [reply core/modules/parsers/audioparser.js]: Hi @donmor apologies for being unclear. I'm pointing out an alternative way of passing the style properties to the domMaker. As you can see from the source, it supports a "style" property on the options object for passing a hashmap of styles:

https://github.com/Jermolene/TiddlyWiki5/blob/master/boot/boot.js#L159-L184

Comment (donmor) [reply core/modules/parsers/audioparser.js]: But the only way I know to insert a video tiddler to another tiddler is using transclusion like:
`{{video.mp4}}`
and the video inserted keeps its original size.
And then, how can I make the video fit the tiddler?

Comment (Jermolene) [reply core/modules/parsers/audioparser.js]: Hi @donmor the change I am suggesting is just a different way of assigning the same style properties to the element.

Comment (donmor) [reply core/modules/parsers/audioparser.js]: You mean...put the style things to another position?

Comment (Jermolene) [reply core/modules/parsers/audioparser.js]: Yes, it's just a different way of passing things to domMaker.

Comment (donmor) [reply core/modules/parsers/audioparser.js]: I just tried your code but not working. The video inserted still keeps its original size.

Comment (Jermolene) [reply core/modules/parsers/audioparser.js]: Hi @donmor apologies, that's entirely my mistake, the syntax I'd suggested doesn't work here.

Comment (donmor) [reply core/modules/parsers/audioparser.js]: You mean I should keep the previous code?

Comment (donmor) [reply core/modules/parsers/audioparser.js]: To be honest I think the previous code "as plain as texts":)

**Comment by Jermolene** [comment]
Hi @donmor apologies for the slow progress!

**Pull request merged**